### PR TITLE
APM-357141 Enable GKE API if it is not enabled

### PR DESCRIPTION
**Pull request by joaquinfilipic-dynatrace**
During installation of autopilot cluster, check if the GKE API needs to be enabled

### CODE REVIEW

File: scripts/deploy-helm.sh [hunk 1]
```diff
@@ -359,6 +359,11 @@ if [[ $CREATE_AUTOPILOT_CLUSTER == "Y" ]]; then
     err "Default region not set. Set default region by running 'gcloud config set compute/region <REGION>'."
     exit 1
   fi
+  CONTAINERS_API=$(gcloud services list --enabled --filter=container.googleapis.com --project="$GCP_PROJECT")
+  if [[ -z "$CONTAINERS_API" ]]; then
+    debug "Enabling Containers API"
```
Comment (equardo) [inline scripts/deploy-helm.sh:364]: Shouldn't this be Kubernetes Engine API?

Comment (joaquinfilipic-dynatrace) [reply scripts/deploy-helm.sh:364]: Yes, that's the name Google uses for the service:
https://cloud.google.com/kubernetes-engine/docs/reference/rest/#:~:text=Kubernetes%20Engine%20API%20Builds%20and%20manages%20container-based,applications%2C%20powered%20by%20the%20open%20source%20Kubernetes%20technology.

Comment (equardo) [reply scripts/deploy-helm.sh:364]: I mean that the debug message says "Enabling Containers API"

**Review by mswiatkowska — APPROVED** [review]

File: scripts/deploy-helm.sh (hunk 1 above)
Comment (joaquinfilipic-dynatrace) [reply scripts/deploy-helm.sh]: Ok, changed to be more accurate

**Review by equardo — APPROVED** [review]

**Pull request merged**